### DISCUSSION TITLE
Registration of new players

### DISCUSSION THREAD

**Discussion by Virtical** [plan-player-analytics/Plan#3776] [Q&A] [answered]
Hello, how can I obtain a **list** of registered players associated with a **specific IP address**?

**Comment by AuroraLS3** [comment]
Plan does not store the IP address of players. You can `grep <ip> *` in a folder with your server log files to find any mentions of the IP in the player join messages.

**Reply by Virtical** [reply]
I'm sorry, I meant to ask about the server address that the player used during registration, not the player's IP address.

**Reply by AuroraLS3** [reply, marked as answer]
You can go to Make a Query and add a Join Addresses filter as well as Registered Between filter
